### ISSUE TITLE
 No library content is displayed. Only currenty playing song is displayed.

### ISSUE THREAD

**Issue by kenden** [abarisain/dmix#765]
MPDroid 1.07.2 Final ba27d2a-c (54)
Android 5.1.1 (Nexus 5)
Using Quodlibet 3.4.1 on Linux Mint 17.2
Server Information:
version: 0.17.0

After installing, I enter my default connection settings: host and port.
When I get back to the main GUI, I have the Artists view selected. No artists are displayed.
I just see a song at the bottom. I press play, the song plays on my PC. Pressing next plays the next song, again, the next, then the next album etc. The album covers are displayed.
But selecting Albums, Playlist, Streams displays the same, nothing.

In Server Information, I also see
Artists: 1
Albums: 1
Songs: 1
In have a few albums in Quodlibet (MPD server) though.

**Comment by avuton** [comment]
Please try the beta, I've seen nothing like what you describe. Also, it's be worth trying the standard MPD implementation.

**Comment by kenden** [comment]
![screenshot_2015-09-30-00-25-00](https://cloud.githubusercontent.com/assets/480586/10180267/ef984466-670b-11e5-9a70-7861f4cfda12.png)

Thanks for the answer. This is a screenshot of the behavior.
I just tried another MPD Android client and get the same behavior, so I think the problem might be more on the server side.
Will try with the official 'mpd' deamon.
Closing as it is likely not a MPDandroid issue.
